src/Microsoft.Fx.Portability/CompressedHttpClient.cs

```diff
@@ -3,6 +3,7 @@
 
 using Microsoft.Fx.Portability.Reporting.ObjectModel;
 using Microsoft.Fx.Portability.Resources;
+using Newtonsoft.Json;
```

Comment:
I thought CodeFormatter would remove all the unused usings.. but it does not. I removed those.